### DISCUSSION TITLE
[Icon request] Addition for Fortran

### DISCUSSION THREAD

**Discussion by nbehrnd** [ryanoasis/nerd-fonts#925] [Ideas]
As the title says, I would like to suggest the addition of an icon for Fortran.

Though contemporary Fortran source code uses `.f90` as file extension marking the *free form* still in ongoing development (e.g., standard Fortran 2018) vs. the *fixed form* of FORTRAN 77 with file extension `.f`, the capitalized «F» is as easy recognizable as are the two snakes in the icon for Python.  It is used e.g., as favicon by [fortran-lang.org](https://fortran-lang.org/), [FortranWiki](https://fortranwiki.org/fortran/show/HomePage), [Fortran *on* Wikipedia](https://en.wikipedia.org/wiki/Fortran), [exercism.org](https://exercism.org/tracks), [Tiobe index](https://www.tiobe.com/tiobe-index/) to mention a few examples.

As checked today on https://www.nerdfonts.com/cheat-sheet, neither `Fortran`, nor `FORTRAN` display a corresponding icon.

To facilitate the creation of this additional icon, I may indicate that there is a .svg file in suitable shape/proportions on [Wikipedia](https://en.wikipedia.org/wiki/File:Fortran_logo.svg).  The creator and copyright owner, @jacobwilliams, released this work to the public domain with the permissive Creative Commons CC0 1.0 license.

preview:

![255px-Fortran_logo svg](https://user-images.githubusercontent.com/19709084/189397707-3b04d868-db30-4499-9c22-d612166d1453.png)

**Comment by Finii** [comment]
https://github.com/ryanoasis/nerd-fonts/discussions/716#discussioncomment-4402601

> @alsimoneau: file-icons would cover https://github.com/ryanoasis/nerd-fonts/discussions/925

**Comment by nbehrnd** [comment]
Thank you for the successful addition of the Fortran icon, which nicely complements managing with source code of other languages (example below for a session in vifm):

![01_cut](https://github.com/ryanoasis/nerd-fonts/assets/19709084/2ae1e203-4912-4ca0-9277-85335ee06bf3)

The addition of the icon was shared with [fortran-lang](https://fortran-lang.discourse.group/t/modern-fortran-logo/2075/30).  Hence I close the ticket as successfully resolved.
